pkg/uprobetracer/tracer.go

```diff
@@ -167,32 +167,20 @@ func (t *Tracer[Event]) AttachProg(progName string, progType ProgType, attachTo
 }
 
 func (t *Tracer[Event]) searchForLibrary(containerPid uint32) ([]string, error) {
-	var targetPaths []string
-	var securedTargetPaths []string
+	var unsecurerTargetPaths []string
```

Comment:
in the function attach you use `unsecured` as prefix insted of `insecurer`, as you use here. Is this a typo or on purpose? The former seems nicer to me.